### ISSUE TITLE
Extension host has with "ENOENT: no such file or directory"

### ISSUE THREAD

**Issue by ChadMcCaffery** [phil294/git-log--graph#70]
I noticed several errors in extension host log, so used extension bisect command to find the issue is caused by this extension.
Looks like path concatenation gone wrong:

```text
2024-01-16 17:15:29.827 [error] ENOENT: no such file or directory, realpath 'C:\d:\git\myrepo\main\output.log': Error: ENOENT: no such file or directory, realpath 'C:\d:\git\myrepo\main\output.log'
```

Maybe my "workspace" is a contributing factor. I work in several repos and multiple branches in each, so I use a folder structure with git worktrees, e.g.:

<pre>
d:\git\<i>repo-name</i>    root for all worktrees
+- \dev             folder for "dev" branch
+- \main            folder for "main" branch
<i>etc.</i>
</pre>

Then I open the root folder in VS Code.

**Comment by eikowagenknecht** [comment]
Same here, plus some `2024-02-15 10:21:12.204 [error] ENOENT: no such file or directory, realpath 'C:\Users\mail\AppData\Local\Programs\Microsoft VS Code\rendererLog': Error: ENOENT: no such file or directory, realpath 'C:\Users\mail\AppData\Local\Programs\Microsoft VS Code\rendererLog'` if you select text in the output window.

**Comment by phil294** [comment]
> so used extension bisect command to find the issue is caused by this extension.

ugh, why are there no extension IDs provided in the host log?!

But I do understand both of you correct that you have not experienced any issues with the extension itself, only noticed the output errors, right?

If that is so, then I'm quite confident I have fixed these problems now with d50729e (technical explanation there) and v0.1.14. I'll close this, please let me know if it happens again! Any details on how/when exactly this happens would be great too.

As to why there was a `C:\d:`, I have no idea
